api/scenario_api_utils/scripts/ros_time_publisher.py

```diff
@@ -1,44 +1,49 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
-
-import rospy
+import rclpy
+from rclpy.node import Node
 import time
 from rosgraph_msgs.msg import Clock
 
-
-class RosTimePublisher():
-
+class RosTimePublisher(Node):
     def __init__(self):
-        self.time_rate_ = rospy.get_param("~time_rate", 1.0)
-        # rospy does not have WallTime)
-        self.initial_time_ = rospy.Time.from_sec(time.time())
+        super().__init__('ros_time_publisher')
+        use_sim_time_param = rclpy.parameter.Parameter(
```

Comment:
I didn't see this in the ROS2 tutorials. I thought for CPP nodes, this parameter is automatically declared. Isn't that true for python as well? Did you check this publisher with simulation and it didn't work properly with declaring `using _sim_time`?

Reply:
removed use_sim_time parameter settings